tests/regression/classification/test_classification.py

```diff
@@ -46,6 +46,7 @@ class TestRegressionMultiClassClassification:
 
     TRAIN_PARAMS = ["--learning_parameters.num_iters", REGRESSION_TEST_EPOCHS]
     templates = Registry(f"src/otx/algorithms/{REG_CATEGORY}").filter(task_type=TASK_TYPE.upper()).templates
+    templates = templates[:1]
```

Comment:
It seems that only one template will be tested. Is it intended?

Reply:
@eunwoosh thanks for your finding! i have to remove that line :)